stacks/kafka-flink/backlog.rst

```diff
@@ -9,10 +9,8 @@ Functionality
 
 Outlook
 =======
-- [o] Currently, with the job(s) at https://github.com/crate/cratedb-flink-jobs,
-  we need Flink 1.13 max., higher versions are incompatible.
-- [o] Update to Flink 1.16. Newer releases of Flink need corresponding Catalog- and
-  Dialect-implementations for CrateDB. We are currently looking into this.
+- [o] Is there an offering also running Apache Kafka and Apache Flink in the Cloud?
+  => Aiven for Apache Flink® -- https://aiven.io/flink
```

Comment:
There are several others, do we want to mention those 2 as well?
https://www.confluent.io/blog/cloud-kafka-meets-cloud-flink-with-confluent-and-immerok/
https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-flink.html

Reply:
Thanks. I had already added Confluent in my tree, but I've also added your link.